server/routes/probationPractitionerReferrals/probationPractitionerReferralsController.ts

```diff
@@ -59,7 +60,11 @@ export default class ProbationPractitionerReferralsController {
     const { accessToken } = res.locals.user.token
 
     const existingDraftReferrals = await this.interventionsService.getDraftReferralsForUserToken(accessToken)
-    const presenter = new FindStartPresenter(existingDraftReferrals)
+
+    const downloadFileInfo = await FileUtils.fileInformation(
+      'assets/downloads/Structured interventions list v1_3_3.xlsx'
+    )
+    const presenter = new FindStartPresenter(existingDraftReferrals, downloadFileInfo)
```

Comment:
It feels a bit off for me to have a controller passing a formatted string to a presenter. Perhaps it would be more appropriate for the controller to be responsible for fetching the file's size as a `number` of bytes, and then the presenter would do the formatting.